src/lib/y2storage/resize_info.rb

```diff
@@ -32,8 +33,33 @@ module Y2Storage
   # This is a wrapper for Storage::ResizeInfo
   class ResizeInfo
     include StorageClassWrapper
+    include Yast::I18n
+    extend Yast::I18n
     wrap_class Storage::ResizeInfo
 
+    # rubocop:disable Metrics/LineLength
+    REASON_TEXTS =
+      {
+        RB_RESIZE_NOT_SUPPORTED_BY_DEVICE:       N_("Resizing is not supported by this device."),
+        RB_MIN_MAX_ERROR:                        N_("Combined limitations of partition and filesystem prevent resizing."),
+        RB_SHRINK_NOT_SUPPORTED_BY_FILESYSTEM:   N_("This filesystem does not support shrinking."),
+        RB_GROW_NOT_SUPPORTED_BY_FILESYSTEM:     N_("This filesystem does not support growing."),
+        RB_FILESYSTEM_INCONSISTENT:              N_("Filesystem consistency check failed."),
+        RB_MIN_SIZE_FOR_FILESYSTEM:              N_("This filesystem already has the minimum possible size."),
+        RB_MAX_SIZE_FOR_FILESYSTEM:              N_("This filesystem already has the maximum possible size."),
+        RB_FILESYSTEM_FULL:                      N_("The filesystem is full."),
+        RB_NO_SPACE_BEHIND_PARTITION:            N_("There is no space behind this partition."),
+        RB_MIN_SIZE_FOR_PARTITION:               N_("This partition already has the minimum possible size."),
```

Comment:
I have problem to recognize what is difference from filesystem full. So this happen when filesystem metadata needs this size?

Reply:
This if for the partition; the other one was for the filesystem. That's two different things. Plus, there might also be an encryption layer in between (or, in very pathological cases, several nested ones).

Reply:
Yes, I am just trying to comment it from user side ( user who understand a bit storage, but is not expert).